services/submission/src/main/java/app/coronawarn/server/services/submission/validation/ValidSubmissionPayload.java

```diff
@@ -84,13 +87,42 @@ public boolean isValid(SubmissionPayload submissionPayload, ConstraintValidatorC
       List<TemporaryExposureKey> exposureKeys = submissionPayload.getKeysList();
       validatorContext.disableDefaultConstraintViolation();
 
-      return checkStartIntervalNumberIsAtMidNight(exposureKeys, validatorContext)
+      boolean isValidPayload = checkStartIntervalNumberIsAtMidNight(exposureKeys, validatorContext)
           && checkKeyCollectionSize(exposureKeys, validatorContext)
           && checkOriginCountryIsValid(submissionPayload, validatorContext)
           && checkVisitedCountriesAreValid(submissionPayload, validatorContext)
           && checkRequiredFieldsNotMissing(exposureKeys, validatorContext)
           && checkTransmissionRiskLevelIsAcceptable(exposureKeys, validatorContext)
           && checkDaysSinceOnsetOfSymptomsIsInRange(exposureKeys, validatorContext);
+
+      if (!isValidPayload) {
+        String payloadString = generatePrintablePayload(submissionPayload);
+        logger.error("Errors caused by invalid payload \n {}", payloadString);
```

Comment:
`\n` will be removed by our log4j config... all logs should be in one single line only